packages/cozy-harvest-lib/src/helpers/konnectors.spec.js

```diff
@@ -63,6 +63,30 @@ describe('Konnectors Helpers', () => {
       )
     })
 
+    it('should should build the default path when account has identifier', () => {
+      expect(
+        buildFolderPath(fixtures.konnector, {
+          ...fixtures.account,
+          identifier: 'login',
+          auth: {
+            login: 'ada@lovelace.com'
+          }
+        })
+      ).toEqual('/Administrative/Test Konnector/ada@lovelace.com')
+    })
+
+    it('should should build the default path when account has identifier', () => {
+      expect(
+        buildFolderPath(fixtures.konnector, {
+          ...fixtures.account,
+          identifier: 'login',
+          auth: {
+            login: 'identifier/with/slashes'
+          }
+        })
+      ).toEqual('/Administrative/Test Konnector/identifier-with-slashes')
+    })
+
```

Comment:
i think this commit should be squashed with the next one?

Reply:
Yes :)